go/README.md

```diff
@@ -1 +1,177 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Tooling
+
+- Visual Studio Code
+- delve
+- golangci-lint
+- staticcheck
+- modd
+
+## Modules
+
+go mod init bitbucket.org/aanzeeonline/[yourprojectname]
+
+Working with a local fork of a module, put an extra line in the go.mod:
+// replace bitbucket.org/aanzeeonline/libgoaz => ../LibGOAZ
+(and remove the comments)
+
+Before pushing the final update to git, make sure that the fork isn't in use anymore
```

Comment:
Standard behavior of go mod places the dependencies in your $GOPATH, go mod vendor puts them in a vendor folder inside your project, running go build -mod vendor uses the dependency put in the vendor folder. This further isolates the project from the globally installed dependencies

Reply:
look at 1.13